third-party/chromium-webtests/webtests/http/tests/devtools/lighthouse/lighthouse-emulate-run-expected.txt

```diff
@@ -25,5 +25,5 @@ ViewportDimensions: {
 }
 
 
-content-width: pass 
+content-width: pass undefined
```

Comment:
hmm?
![image](https://user-images.githubusercontent.com/4071474/93530076-94cdfd80-f902-11ea-87ae-66e9e2203501.png)

Reply:
`audit.explanation` is optional, so can be `undefined` :)

It was the empty string before...not sure that's better :)

Reply:
seems like we should only put `audit.explanation` into the result when the audit is failing 🤔

Reply:
can you add to the list? I also want to pull this into a fn in LighthouseTestRunner (also used in other tests)